daemon/api.go

```diff
@@ -257,6 +258,12 @@ var (
 		POST:     ackWarnings,
 	}
 
+	modelCmd = &Command{
+		Path: "/v2/model",
+		POST: postModel,
+		// FIXME: provide getModel here instead of via debug?
```

Comment:
that probably makes more sense tbh (but I don't really mind)

Reply:
yes, I agree later we likely want  that and to support:

snap model
snap model --serial

but we need to think a bit what their output (variants) is and what is the response format of the API endpoints, so better not to rush there.